src/intersection-observer-polyfill.js

```diff
@@ -273,6 +284,19 @@ export class IntersectionObserverPolyfill {
       }
     }
 
+
+    // Add a mutation observer to tick ourself
+    if (MutationObserver && !this.mutationObserver_) {
```

Comment:
This will throw an error if `MutationObserver` is not defined. Use `win.MutationObserver`